proto/source_metadata.proto

```diff
@@ -66,6 +66,7 @@ message Confluence {
   string location = 7;
   string file = 8;
   string user = 9;
+  string commentId = 10;
```

Comment:
Can we follow the existing `snake_case` here?

Reply:
Good catch. Fixed it.